### ISSUE TITLE
msgfmt: found 1 fatal error

### ISSUE THREAD

**Issue by jvalleroy** [freedombox/Plinth#1039]
I get this error when running ```vagrant provision```:
```
==> default: django.core.management.base.CommandError: Execution of msgfmt failed: /vagrant/plinth/locale/de/LC_MESSAGES/django.po:542: 'msgstr' is not a valid Python format string, unlike 'msgid'. Reason: In the directive number 1, the character 'b' is not a valid conversion specifier.
```

It's complaining about this translated string:
```
#: plinth/modules/disks/views.py:93
#, python-format, python-brace-format
msgid ""
"Warning: Low space on system partition ({percent_used}% used, {free_space} "
"free)."
msgstr ""
"Warnung: Geringer Speicherplatz auf der Systempartition ({percent_used}% "
"belegt, {free_space} verfügbar)."
```

I guess the problem is the "% b".

**Comment by jvalleroy** [comment]
Another related problem:
```
==> default: django.core.management.base.CommandError: Execution of msgfmt failed: /vagrant/plinth/locale/tr/LC_MESSAGES/django.po:539: number of format specifications in 'msgid' and 'msgstr' does not match
```

```
#: plinth/modules/disks/views.py:93
#, python-format, python-brace-format
msgid ""
"Warning: Low space on system partition ({percent_used}% used, {free_space} "
"free)."
msgstr ""
"İkaz: sistem disk bölümünde düşük alan ({percent_used} kullanıldı, "
"{free_space} boş)."
```

**Comment by JoKeyser** [comment]
Is this in `master`? There, I can't reproduce this exactly - instead I get this:
```
==> default: django.core.management.base.CommandError: Execution of msgfmt failed: /vagrant/build/lib/plinth/locale/nb/LC_MESSAGES/django.po:419: 'msgstr' is not a valid Python format string, unlike 'msgid'. Reason: In the directive number 4, the character '(' is not a valid conversion specifier.
==> default: msgfmt: found 1 fatal error
```

Also, `plinth/modules/disks/views.py` does not exist in current master anymore. And in branch v.0.15.x, I don't reproduce this...

**Comment by JoKeyser** [comment]
One common oddity in both problems is they declare both format types (whatever they actually mean): `#, python-format, python-brace-format`. If I erase the first, i.e. set to `#, python-brace-format`, then the error doesn't exist anymore. But I'm not sure what this means, nor where best to fix it (in Weblate?).

**Comment by JoKeyser** [comment]
Oh, and in Weblate, the strings are still listed with origin `plinth/modules/disks/views.py:93`, which may be part of the problem?

**Comment by jvalleroy** [comment]
@JoKeyser I updated the strings, so they should point to the correct source location now.

**Comment by JoKeyser** [comment]
@jvalleroy , great. Did you also remove the "bad" flag? In my view, both are still unchanged:
![wrongweblateflag](https://user-images.githubusercontent.com/8722846/31055817-41873744-a6b8-11e7-9e86-28188f592f31.png)

**Comment by jvalleroy** [comment]
@JoKeyser It is generated with both flags when I run "setup.py update_translations". So we need to find a way to have it generate with only the correct flag. (I wouldn't want to manually modify it every time.)

**Comment by JoKeyser** [comment]
Turns out it's a known, hairy issue, with a workaround: https://code.djangoproject.com/ticket/27221, see PR #1041

**Comment by jvalleroy** [comment]
Fixed in #1041.
